views/sms/index.js

```diff
@@ -241,7 +241,7 @@ exports.respond = function(req, res) {
             return save_utils.countRequestsPerRegion(region_code);
         }).then( function(numRequests) {
             requestData = save_utils.createSerial(numRequests, req.cookies.request_object, region_code);
-            requestData.is_sms = 'sms';
+            requestData.origin = 'sms';
```

Comment:
Okay, actually, it's more complicated, and some of this can be done in this change and some of it should not.

There are a couple kinds of "origin" or "source," which is why this is so confusing -- thanks for catching it, @kfogel.  There's the conceptual source of the request -- did it come from an SMS, the regular web form, or the 311 web form?  Then there's the URL of the request, specifically, which pages use to determine which elements to display or not display.  Changing *that* is a bigger change, outside the scope of this PR. (I discovered that as I was trying to make it.)  But I think we can call the "conceptual" source the `source` everywhere, and I'll make that change in this PR. 👍